CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.4.0
+
+-   expose new IQL query type via the Connector API
```

Comment:
you can be more precise and tell what routes have been added and what they can be used for

Reply:
additionally please tell about the used runtime version here.